draft-ietf-quic-recovery.md

```diff
@@ -1094,7 +1094,9 @@ max_ack_delay:
 : The maximum amount of time by which the receiver intends to delay
   acknowledgments for packets in the Application Data packet number space. The
   actual ack_delay in a received ACK frame may be larger due to late timers,
-  reordering, or lost ACK frames.
+  reordering, or lost ACK frames. max_ack_delay is initialized to 0 and
+  updated when transport parameters are exchanged. If a peer does not
+  specify max_ack_delay, it is set to 25ms.
```

Comment:
As @martinthomson points out, this is a deviation from the definition of max_ack_delay in the transport draft, that says that the value defaults to 25ms and is updated when receiving the transport parameters from the peer.

We could have different definitions, but IMO it is unnecessary in this case, based on our agreement that persistent congestion is a connection-wide event and that it can be applied during the handshake.

Reply:
Good point.  How about a reference to the section max_ack_delay is discussed in transport instead of this change?

Reply:
@ianswett That SGTM.

Reply:
I'd suggest some text like this, to avoid repeating the details here.
```
The maximum amount of time by which the receiver intends to delay acknowledgments
for packets in the Application Data packet number space, as defined by the eponymous
transport parameter (Section 18.2 of {{QUIC-TRANSPORT}}). Note that the actual
ack_delay in a received ACK frame may be larger due to late timers, reordering, or lost
ACK frames.
```